src/content/docs/data-apis/custom-data/aws-privatelink.mdx

```diff
@@ -619,6 +701,19 @@ These are the New Relic endpoint services available via AWS PrivateLink:
             `com.amazonaws.vpce.eu-central-1.vpce-svc-06d5b2d7e79ddd78e`
             </td>
           </tr>
+          <tr>
+            <td>
+
+            </td>
```

Comment:
@haus Same with this one?

Reply:
same comment as above. the cell is intentionally blank to imply a continuation of the product from the row immediately above. i copied that pattern from the us-east-2 table